### ISSUE TITLE
I get diffs for supported_scopes in paln after apply.

### ISSUE THREAD

**Issue by syokubai** [authlete/terraform-provider-authlete#86]
Hi there,

I'm not familiar with English, but thank you in advance.
If I apply terraform and then plan terraform again, I get a diff in supported_scopes.
If I change the order of supported_scopes and apply, the diff will not appear.
However, it is difficult to write tf files with the order in mind, is there any way to avoid the diff?

### Terraform Version
```
Terraform v1.3.6
on linux_amd64
+ provider registry.terraform.io/authlete/authlete v1.2.1
```

### Affected Resource(s)
- authlete_service

### Terraform Configuration Files
```hcl
terraform {
  required_providers {
    authlete = {
      source  = "authlete/authlete"
      version = "1.2.1"
    }
  }
}

provider "authlete" {
  # Configuration options
  service_owner_key    = xxx
  service_owner_secret = "xxx"
}

resource "authlete_service" "my_authlete_service" {
  issuer                   = "https://as.mydomain.com"
  service_name             = "MyDomainAS"
  description              = "A terraform based service for managing the Authlete based OAuth server"
  supported_grant_types    = ["AUTHORIZATION_CODE"]
  supported_response_types = ["CODE"]
  supported_scopes {
    name = "test2"
  }
  supported_scopes {
    name = "test1"
  }
}

```

### Expected Behavior
Nothing diffs in terraform plan after terraform apply.

### Actual Behavior
I get the following diff in terraform plan after terraform apply  
```
  # authlete_service.my_authlete_service will be updated in-place
  ~ resource "authlete_service" "my_authlete_service" {
        id                                            = "60742397172544"
        # (69 unchanged attributes hidden)

      ~ supported_scopes {
          ~ name          = "test1" -> "test2"
            # (1 unchanged attribute hidden)
        }
      ~ supported_scopes {
          ~ name          = "test2" -> "test1"
            # (1 unchanged attribute hidden)
        }
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

### Steps to Reproduce
1. `terraform apply`
2. `terraform plan`

### Important Factoids
When I use the following order of supported_scopes, there will be no diff
```
resource "authlete_service" "my_authlete_service" {
  issuer                   = "https://as.mydomain.com"
  service_name             = "MyDomainAS"
  description              = "A terraform based service for managing the Authlete based OAuth server"
  supported_grant_types    = ["AUTHORIZATION_CODE"]
  supported_response_types = ["CODE"]
  supported_scopes {
    name = "test1"
  }
  supported_scopes {
    name = "test2"
  }
}
```

**Comment by hefa** [comment]
@OhkiRyuma This issue is fixed in version 1.3.0. Thank you again for letting us know about the problem.

**Comment by OhkiRyuma** [comment]
@hefa   
Thanks for the quick response!   
It's so much easier to use.
